network_function/README.md

```diff
@@ -7,7 +7,7 @@ This provides a "transparent pipe" which can be extended to
 provide true "bump in the wire" network functions.
 This code supports packets up to 64K.
 
-To compile 
+**To compile**
```

Comment:
use code blocks for what you expect to be run/executed

Reply:
updated